community/slack_channel_guide.md

```diff
@@ -21,6 +21,7 @@ Want to add/change something? Give a PR.
 | #blockchain | Talk about blockchain technology (but not crypto trading) |
 | #career_advice | Ask for a resume review, career advice, questions about job offers |
 | #coding-interviews | Get help on algorithims and coding problems for job interviews or fun |
+| #cooking  | Recipes, cooking techniques, general food chat |
```

Comment:
Weren't we going to break this and others out into a brand new section called "Life Hacks"?

That section will include  #mental-health #music #parenting #outdoors #homeimprovement #wellness and I think #cooking makes sense in there too